app/ui-account/client/accountProfile.js

```diff
@@ -416,11 +416,10 @@ Template.accountProfile.events({
 		const password = instance.password.get();
 
 		const send = $(e.target.send);
-		send.addClass('loading');
 		const reqPass = ((email !== getUserEmailAddress(user))
 			|| s.trim(password)) && (user && user.services && user.services.password && s.trim(user.services.password.bcrypt));
 		if (!reqPass) {
-			return instance.save(undefined, () => setTimeout(() => send.removeClass('loading'), 1000));
+			return instance.save();
```

Comment:
Rather than removing completely the loading, just remove the `setTimeout` from the callback, in theory, it should work, since the callback will be only called when `instance.save()` is done.